packages/node-integration-tests/utils/run-tests.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-console */
+import childProcess from 'child_process';
+import os from 'os';
+
+const testPaths = childProcess.execSync('jest --listTests', { encoding: 'utf8' }).trim().split('\n');
```

Comment:
I think having an explanation here is both helpful in its own right and possibly a way to forestall other folks asking the same question Abhi asked about `foreEach`.

```suggestion
// This will serve as a pool of remaining tests from which the threads spawned below can draw
const testPaths = childProcess.execSync('jest --listTests', { encoding: 'utf8' }).trim().split('\n');
```

Also:

 > `jest --listTests`

TIL! 🙂